src/BioSequences.jl

```diff
@@ -227,4 +227,83 @@ include("search/ApproxSearchQuery.jl")
 include("search/re.jl")
 include("search/pwm.jl")
 
+
```

Comment:
Can this be put in the search folder instead of the top level module file?

Reply:
I've put it into the search folder. Let me know if you had different else in mind.